sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/LockRenewalOperationTest.java

```diff
@@ -194,7 +194,7 @@ void cancellation() {
                 operation.close();
             })
             .expectComplete()
-            .verify(renewalPeriod);
+            .verify();
```

Comment:
Will the remove of timeout cause test case never stop?

Reply:
With reference to your suggestion, it has been fixed in the new version.